### ISSUE TITLE
could "search and replace" be used as a "temporary cell filter" option ?

### ISSUE THREAD

**Issue by stonebig** [jupyter/notebook#855]
In some Office suites Spreadsheet, you can use the "search" function to filter cells, like in a split/appl combine:
- `show me all cells with print, so I can check if they respect the standard print () format"
- (doing my parenthesis work on this old Ipython tutorial)
- remove the temporary filter.

**Comment by Carreau** [comment]
Yes it's planed, not through search and replace though. And 4.1 is feature freeze, so likely 5.0 or later.
